packages/framework-provider-azure-infrastructure/src/infrastructure/index.ts

```diff
@@ -44,10 +45,14 @@ async function deployApp(logger: Logger, config: BoosterConfig, rockets?: Infras
 /**
  * Nuke all the resources used in the Resource Group
  */
-async function nukeApp(_logger: Logger, config: BoosterConfig): Promise<void> {
+async function nukeApp(_logger: Logger, config: BoosterConfig, rockets?: InfrastructureRocket[]): Promise<void> {
   const credentials = await azureCredentials()
   const resourceManagementClient = await createResourceManagementClient(credentials)
 
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  rockets?.filter((rocket) => rocket.unmountStack).map((rocket) => rocket.unmountStack())
```

Comment:
Shouldn't you pass the rocket-utils as parameter here too?

Reply:
Not sure about it. right now we don't need unmount anything on Azure as removing the resource group all the children resources are unmounted

Reply:
@gonzalogarciajaubert 
ok! BTW, with regard to the previous comment, maybe you can avoid that Typescript complain by doing it this way instead:
```suggestion
  rockets?.forEach((rocket) => rocket.unmountStack?.(rocketUtils))
```
That's how it is done in AWS and we got no complains 😃

Reply:
Awesome!